staking/programs/staking/src/state/positions.rs

```diff
@@ -1385,6 +1409,13 @@ pub mod tests {
             {
                 return false;
             }
+
+            if (target == &TargetWithParameters::Voting && curr_state == &PositionState::UNLOCKED)
```

Comment:
this tests that slashing ignores unlocked positions

Reply:
do we need brackets here?